apps/shared-components/badge/badge.component.html

```diff
@@ -8,7 +8,7 @@
   class="{{ position }} {{ color }}"
 >
   <div>
-    <p [ngClass]="{ 'small-text': fontSize == 'small' }" class="{{ fontColor }}">
+    <p [ngClass]="{ 'small-text': fontSize == 'small', 'mini-text': fontSize == 'mini' }" class="{{ fontColor }}">
```

Comment:
let's make mini to extra-small or xs

Reply:
xtra small is already made with 10px font size, its 8px font size